### PR TITLE
Volume-changes

### PR DESCRIPTION
**Pull request by Lamandus**
#### Summary
SUMMARY: Balance "Change Volume on a lot of stuff"

#### Purpose of change
This is a "port" of all my changes I made to DDA, I want to provide them to you guys, since I have not a big interest in getting back into DDA anytime soon.

#### Describe alternatives you've considered
Going back...

#### Testing
Works

### CODE REVIEW

**Comment by Coolthulhu** [comment]
Some notes:
* Round weights are better than arbitrary, since most containers have volumes that round up to 250ml
* In BN, volume is also abstracted "bulkiness", so its value should be between actual volume (as measured by amount of water displaced if it was submerged) and volume of its "bounding box"

**Comment by Lamandus** [comment]
> Some notes:
> 
> * Round weights are better than arbitrary, since most containers have volumes that round up to 250ml
> * In BN, volume is also abstracted "bulkiness", so its value should be between actual volume (as measured by amount of water displaced if it was submerged) and volume of its "bounding box"

no problem, I will round up the stuff a bit.

**Comment by Lamandus** [comment]
is the failed check something I can fix?

**Comment by Coolthulhu** [comment]
> is the failed check something I can fix?

Yes, looks like in this case it's actually relevant to the PR.
Firefighter profession can't put an item it has onto the belt. You should try loading a game with your changes and creating a fireman character to see which item won't fit.

File: data/json/items/resources/metal.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "material": [ "steel" ],
     "qualities": [ [ "HAMMER", 1 ] ],
     "techniques": [ "WBLOCK_1" ],
-    "volume": "1 L",
+    "volume": "400 ml",
```
Comment (Coolthulhu) [inline data/json/items/resources/metal.json:15]: I'd say round it up to 0.5L.

File: data/json/items/resources/metal.json [hunk 2]
```diff
@@ -65,7 +65,7 @@
     "symbol": "/",
     "material": [ "copper" ],
     "techniques": [ "WBLOCK_1" ],
-    "volume": "500 ml",
+    "volume": "400 ml",
```
Comment (Coolthulhu) [inline data/json/items/resources/metal.json:68]: This one could be thinner, as per the description. 250ml would work.

File: data/json/items/tool/electronics.json [hunk 3]
```diff
@@ -467,7 +467,7 @@
     "name": { "str": "handheld game system" },
     "description": "This is a portable games console in working condition, with a backlit screen allowing you to play in the dark.  You can use it to play it for a little while, but this requires batteries.",
     "weight": "200 g",
-    "volume": "500 ml",
+    "volume": "350 ml",
```
Comment (Coolthulhu) [inline data/json/items/tool/electronics.json:470]: I'd round it down to 250ml. Smartphone is 100ml.

File: data/json/items/tool/entry_tools.json [hunk 4]
```diff
@@ -4,8 +4,8 @@
     "type": "TOOL",
     "name": { "str": "crowbar" },
     "description": "This is a hefty prying tool.  Use it to open locked doors without destroying them or to lift manhole covers.  You could also wield it to bash some heads in.",
-    "weight": "500 g",
-    "volume": "1 L",
+    "weight": "1200 g",
+    "volume": "250 ml",
```
Comment (Coolthulhu) [inline data/json/items/tool/entry_tools.json:8]: That's very little. What is it based on?

File: data/json/items/tool/entry_tools.json [hunk 5]
```diff
@@ -60,7 +60,7 @@
     "name": { "str": "makeshift crowbar" },
     "description": "This is a pipe whose ends have been bent and hammered flat to resemble a crowbar.  Use it to open locked crates without destroying them, or to lift manhole covers.  You could also wield it to fight with, in a pinch.",
     "weight": "1250 g",
-    "volume": "1 L",
+    "volume": "400 ml",
```
Comment (Coolthulhu) [inline data/json/items/tool/entry_tools.json:63]: As pipe above.

File: data/json/items/tool/firefighting.json [hunk 6]
```diff
@@ -146,7 +146,7 @@
     "techniques": [ "WBLOCK_1" ],
     "flags": [ "DURABLE_MELEE", "STAB", "SHEATH_AXE" ],
     "weight": "2900 g",
-    "volume": "1900 ml",
+    "volume": "400 ml",
```
Comment (Coolthulhu) [inline data/json/items/tool/firefighting.json:149]: Judging by other stats, it seems to be a larger "variant" of this item.

File: data/json/items/tool/workshop.json [hunk 7]
```diff
@@ -124,7 +124,7 @@
     "name": { "str": "paint chipper" },
     "description": "A tool similar to a chisel, designed to remove paint.",
     "weight": "60 g",
-    "volume": "500 ml",
+    "volume": "300 ml",
```
Comment (Coolthulhu) [inline data/json/items/tool/workshop.json:127]: Rounding to 250ml is better than 300ml. Same for ones below.

File: data/json/items/tool/workshop.json [hunk 8]
```diff
@@ -791,7 +791,7 @@
     "name": { "str": "soldering iron" },
     "description": "This is a device with a metal tip that can get very hot.  It is necessary for advanced electronics crafting.  You could also use it to cauterize wounds, if you had to.",
     "weight": "181 g",
-    "volume": "500 ml",
+    "volume": "100 ml",
```
Comment (Coolthulhu) [inline data/json/items/tool/workshop.json:794]: It's not just a soldering iron itself, but also the battery compartment for it.

File: data/json/items/tool/entry_tools.json (hunk 4 above)
Comment (Lamandus) [reply data/json/items/tool/entry_tools.json:8]: Geometry. But I would round it up to 500 to be as bulky as the makeshift counterpart

File: data/json/items/tool/firefighting.json (hunk 6 above)
Comment (Lamandus) [reply data/json/items/tool/firefighting.json:149]: up to 1,5 liter it is!

File: data/json/items/tool/workshop.json (hunk 8 above)
Comment (Lamandus) [reply data/json/items/tool/workshop.json:794]: let's round up to 250 then

**Pull request merged**